### ISSUE TITLE
ControlP5.Range right value label does not receive any font changes from .getValueLabel().setFont(font)

### ISSUE THREAD

**Issue by DavidGoedicke** [sojamo/controlp5#55]
The font of the rightvalueLable in a range slider can only be set by declaring 
`.setFont(font);` before the range slider is created.
 
Changing the font with `.getValueLabel().setFont(font);` does only affect the left value label. 
Meaning that ones it is created its font can never be changed.

Example(extended from the range example):
```processing
cp5 = new ControlP5(this); 
  PFont font = createFont("arial", 17);
  //cp5.setFont(font);
  range = cp5.addRange("rangeController")
    .setBroadcast(false) 
    .setPosition(50, 50)
    .setSize(400, 40)
    .setHandleSize(20)
    .setRange(0, 255)
    .setRangeValues(50, 100)
    .setBroadcast(true)
    .setColorForeground(color(255, 40))
    .setColorBackground(color(255, 40))  
    ;
  cp5.getController("rangeController").getValueLabel().setFont(font);
  cp5.getController("rangeController").getCaptionLabel().setFont(font);
```

produces:
![disatisfying](https://cloud.githubusercontent.com/assets/2676840/13473494/73a1a78a-e06e-11e5-9f1f-d887366ec47e.png)

This is not a pressing issue but it would be great to have fixed for the sake of consistency.

**Comment by sojamo** [comment]
ok, thanks for filing the issue, will be fixed.

**Comment by sojamo** [comment]
fixed with 2.2.6 but use 
`cp5.getController("rangeController").setFont(font)`
